ygen/ir.go

```diff
@@ -495,10 +531,20 @@ type EnumeratedYANGType struct {
 	// generation mechanism to select how different enumerated
 	// value types are output.
 	Kind EnumeratedValueType
+	// TODO(wenbli): This needs to be replaced using a plugin mechanism.
```

Comment:
What does plugin mechanism mean here?

Reply:
I meant the flag mechanism you mentioned here: https://github.com/openconfig/ygot/pull/655#discussion_r860233845